x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/IntArrayVector.java

```diff
@@ -27,6 +27,16 @@ final class IntArrayVector extends AbstractVector implements IntVector {
 
     private final int[] values;
 
+    /**
+     * The minimum value in the block.
+     */
+    private Integer min;
+
+    /**
+     * The minimum value in the block.
+     */
+    private Integer max;
```

Comment:
I'm not tracking the memory of the used `Integer`, just the slot. Maybe I should? Or maybe I should just use `boolean` rather than `null` for "is this set?"

Reply:
It's okay not to track this. We shouldn't have too many blocks at the same time.